### PR TITLE
Polish locale & small style changes

### PR DESCRIPTION
**Pull request by lenisko**
- Add Polish locale \o/
- Resize columns for gym, pokemon & pokestop to fit labels (tested on English as well)

Co-translators: Kasia

### CODE REVIEW

File: src/components/layout/drawer/Drawer.jsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default function Sidebar({
               || category === 'pokestops'
               || category === 'nests')
               && (
-                <Grid item xs={6} style={{ textAlign: 'center' }}>
+                <Grid item xs={7} style={{ textAlign: 'center' }}>
```
Comment (TurtIeSocks) [inline src/components/layout/drawer/Drawer.jsx:102]: I'm not a huge fan of this one as it just looks off centered in non-Polish languages. Maybe some logic specific to Polish language or preferably, something to shrink the size of the font to fit the xs={6} button size.

File: src/components/popups/Pokemon.jsx [hunk 2]
```diff
@@ -427,12 +427,12 @@ const ExtraInfo = ({ pokemon, perms, t }) => {
...
       {[first_seen_timestamp, updated].map((time, i) => (
         <Fragment key={time}>
-          <Grid item xs={5} style={{ textAlign: 'center' }}>
+          <Grid item xs={7} style={{ textAlign: 'center' }}>
             <Typography variant="caption" align="center">
               {i ? t('lastSeen') : t('firstSeen')}:
             </Typography>
           </Grid>
-          <Grid item xs={6} style={{ textAlign: 'right' }}>
+          <Grid item xs={4} style={{ textAlign: 'right' }}>
```
Comment (TurtIeSocks) [inline src/components/popups/Pokemon.jsx:435]: Screen Shot 2021-08-06 at 12 45 50 PM

The pokestop and gyms are fine but this one causes some issues for English.

**Review by TurtIeSocks — CHANGES_REQUESTED** [review]
Awesome work on Polish locales! 

Some minor comments with formatting issues that showed up.

File: src/components/layout/drawer/Drawer.jsx (hunk 1 above)
Comment (lenisko) [reply src/components/layout/drawer/Drawer.jsx:102]: Per-language would be better... but it's a PITA 😂I will remove changes related to grid for now.

File: src/components/popups/Pokemon.jsx (hunk 2 above)
Comment (lenisko) [reply src/components/popups/Pokemon.jsx:435]: Gotcha my mistake. I will remove changes related to styling for now

**Comment by lenisko** [comment]
@TurtIeSocks for now I have removed styling changes

**Review by TurtIeSocks — APPROVED** [review]

**Pull request merged**